src/Connection/Manager.php

```diff
@@ -321,6 +324,8 @@ private function switchNodeIfNeeded()
             $this->nodeId
         );
 
+        $previous = null;
```

Comment:
I know it's already merged, but `$previous` is not a very good variable name, for anything.